rclpy/rclpy/__init__.py

```diff
@@ -32,6 +32,8 @@ def init(*, args=None):
 def create_node(node_name, *, namespace=None):
     namespace = namespace or ''
     failed = False
+    if not _rclpy.rclpy_is_initialized():
```

Comment:
Can't `_rclpy` actually be used in this case since it hasn't been initialized yet? Also if that would be possible the existing function `rclpy_ok` could be used (no need to introduce `rclpy_is_initialized`).